### DISCUSSION TITLE
项目中部署失败了怎么办呢？- [一个示例的QA帖子]

### DISCUSSION THREAD

**Discussion by tangly1024** [tangly1024/NotionNext#66] [Q&A]
留下你的Github地址，以及NotionPageId，我可以为你查看问题～

**Comment by BazingLEE** [comment]
大大，我的gihub地址是https://github.com/BazingLEE/NotionNext，我的NotionPageid是hd001965183884b759d2210fe0fa31832。
我的问题是部署之后只有在手机的谷歌浏览器才能看到博客，其他的途径都无法连接到网站，或者连接网站超时。
